cli/admin/README.md

```diff
@@ -86,6 +86,7 @@ From that point onwards, you can check the available commands by typing `help`:
 > help
 All available commands:
         bk cleanup : Removes orphan BookKeeper Ledgers that are not used by any BookKeeperLog.
+        bk delete-ledgers <log-id> <ledger-id>: Deletes all the ledgers starting with given ledger-id as the starting index.
```

Comment:
This command deletes all the ledgers for a Bookkeeper Log beyond/after the input `ledger-id`, right? If so, please rephrase the description because it could be clearer. Also, specify if the `ledger-id` parameter is "inclusive" or not (does the command also delete the ledger related to `ledger-id` parameter or not?).

Reply:
updated